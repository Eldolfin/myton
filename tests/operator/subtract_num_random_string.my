print(1 - "a") # Error
