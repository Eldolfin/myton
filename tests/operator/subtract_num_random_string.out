[31merror[error]: unsupported operand type(s) for -: 'number' and 'str'
----- Traceback -----
  â”Œâ”€ <unknown>:1:9
  | 
1 | print(1 - "a") # Error
  |          ^
[0m